Tools/module_config/generate_params.py

```diff
@@ -282,12 +282,34 @@ def add_local_param(param_name, param_def):
 
 Note that non-motor outputs might already be active in prearm state if COM_PREARM_MODE is set.
 '''
-        minimum_description = \
-'''Minimum output value (when not disarmed).
-'''
-        maximum_description = \
-'''Maxmimum output value (when not disarmed).
-'''
+        if param_prefix == 'SIM_GZ_SV':
+            minimum_description = \
+    '''
+    Servo output represents angles [0, 360] deg scaled with 10 for numerical precision,
+    with an offset of -180 deg to allow for negative angles.
+    Hence, myMin = (180 + myOffset) * 10 resulting angles within [-180,180] deg.
+
+    Min max must be coherent with joint limits in the airframe/model.sdf
+
+    Ex. ailerons with a minimum angle -45 deg would be 1350.
```

Comment:
@Jaeyoung-Lim I put the explanation here.

If the outputs are ints then we cannot have more than 1 deg precision , so to allow for one decimal the scaling was set to 10.